dracut/30ignition/ignition-cmdline.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+type getarg >/dev/null 2>&1 || . /lib/dracut-lib.sh
+getargbool 0 ignition.firstboot || exit 0
+
+ign_file="/sys/firmware/qemu_fw_cfg/by_name/opt/com.coreos/config/raw"
```

Comment:
I think there's a circular dependency here; you're trying to specify the networking config via Ignition, but Ignition in the general case is fetched over the network.

This qemu path is special in that it's local only.

Reply:
> I think there's a circular dependency here; you're trying to specify the networking config via Ignition, but Ignition in the general case is fetched over the network.

ok, in such case it needs to go directly to kernel cmdline.

> 
> This qemu path is special in that it's local only.